setup.py

```diff
@@ -50,9 +53,9 @@
 setup(
```

Comment:
I quite like having extra project specific links here.

I've got some for django-bleach 
https://github.com/marksweb/django-bleach/blob/main/setup.py#L109

What do you think?  Could do something similar;

```
    project_urls={
        'Documentation': 'https://djangocms-frontend.readthedocs.io/en/latest/',
        'Release notes': 'https://github.com/django-cms/djangocms-frontend/blob/master/CHANGELOG.rst',
        'Issues': 'https://github.com/django-cms/djangocms-frontend/issues',
        'Source': 'https://github.com/django-cms/djangocms-frontend',
    }
```